src/af3cli/__main__.py

```diff
@@ -1081,7 +1081,7 @@ def config(
         self,
         filename: str = DEFAULT_FILENAME,
         jobname: str = "job",
-        version: int = 1,
```

Comment:
This will probably break older AlphaFold installations, since it results in unknown keywords in the input files.